lib/ui/src/components/layout/container.js

```diff
@@ -285,13 +287,35 @@ class Layout extends Component {
     };
   }
 
-  componentDidUpdate() {
+  componentDidUpdate(prevProps, prevState) {
     const { resizerPanel, resizerNav } = this.state;
 
     persistance.set({
       resizerPanel,
       resizerNav,
     });
+    const { width: prevWidth, height: prevHeight } = prevProps.bounds;
+    const { bounds, options } = this.props;
+    const { width, height } = bounds;
+    if (width !== prevWidth || height !== prevHeight) {
```

Comment:
Why can't this block be done in the `getDerivedStateFromProps` ?

Reply:
I want to move the resizer by the delta of boundaries. I could move it to getDeriverStateFromProps by changing the resizerPanel state to store width and height instead of a position. What do you think?

Reply:
yeah, It may work I think.